internal/services/applicationinsights/application_insights_resource.go

```diff
@@ -81,6 +81,7 @@ func resourceApplicationInsights() *pluginsdk.Resource {
 			"workspace_id": {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
+				Computed:     true,
```

Comment:
could we include a comment here with `// NOTE: O+C ` to explain why this should be computed?

Reply:
Updated